code/modules/mob/living/carbon/monkey/life.dm

```diff
@@ -89,620 +90,627 @@
 
-			if (radiation > 100)
-				radiation = 100
-				Weaken(10)
-				src << "<span class='warning'>You feel weak.</span>"
-				emote("collapse")
-
-			switch(radiation)
-				if(1 to 49)
-					radiation--
-					if(prob(25))
-						adjustToxLoss(1)
-						updatehealth()
-
-				if(50 to 74)
-					radiation -= 2
+		switch(radiation)
+			if(1 to 49)
+				radiation--
+				if(prob(25))
 					adjustToxLoss(1)
-					if(prob(5))
-						radiation -= 5
-						Weaken(3)
-						src << "<span class='warning'>You feel weak.</span>"
-						emote("collapse")
 					updatehealth()
 
-				if(75 to 100)
-					radiation -= 3
-					adjustToxLoss(3)
-					if(prob(1))
-						src << "<span class='warning'>You mutate!</span>"
-						randmutb(src)
-						domutcheck(src,null)
-						emote("gasp")
-					updatehealth()
-
-	// Separate proc so we can jump out of it when we've succeeded in spreading disease.
-	proc/findAirborneVirii()
-		if(blood_virus_spreading_disabled)
-			return 0
-		for(var/obj/effect/decal/cleanable/blood/B in get_turf(src))
-			if(B.virus2.len)
-				for (var/ID in B.virus2)
-					var/datum/disease2/disease/V = B.virus2[ID]
-					if (infect_virus2(src,V, notes="(Airborne from blood)"))
-						return 1
-
-		for(var/obj/effect/decal/cleanable/mucus/M in get_turf(src))
-			if(M.virus2.len)
-				for (var/ID in M.virus2)
-					var/datum/disease2/disease/V = M.virus2[ID]
-					if (infect_virus2(src,V, notes="(Airborne from mucus)"))
-						return 1
-		return 0
-
-	proc/handle_virus_updates()
-		if(status_flags & GODMODE)	return 0	//godmode
-		if(bodytemperature > 406)
-			for(var/datum/disease/D in viruses)
-				D.cure()
-			for (var/ID in virus2)
-				var/datum/disease2/disease/V = virus2[ID]
-				V.cure(src)
+			if(50 to 74)
+				radiation -= 2
+				adjustToxLoss(1)
+				if(prob(5))
+					radiation -= 5
+					Weaken(3)
+					src << "<span class='warning'>You feel weak.</span>"
+					emote("collapse")
+				updatehealth()
 
-		src.findAirborneVirii()
+			if(75 to 100)
+				radiation -= 3
+				adjustToxLoss(3)
+				if(prob(1))
+					src << "<span class='warning'>You mutate!</span>"
+					randmutb(src)
+					domutcheck(src,null)
+					emote("gasp")
+				updatehealth()
 
+// Separate proc so we can jump out of it when we've succeeded in spreading disease.
+/mob/living/carbon/monkey/proc/findAirborneVirii()
+	if(blood_virus_spreading_disabled)
+		return 0
+	for(var/obj/effect/decal/cleanable/blood/B in get_turf(src))
+		if(B.virus2.len)
+			for (var/ID in B.virus2)
+				var/datum/disease2/disease/V = B.virus2[ID]
+				if (infect_virus2(src,V, notes="(Airborne from blood)"))
+					return 1
+
+	for(var/obj/effect/decal/cleanable/mucus/M in get_turf(src))
+		if(M.virus2.len)
+			for (var/ID in M.virus2)
+				var/datum/disease2/disease/V = M.virus2[ID]
+				if (infect_virus2(src,V, notes="(Airborne from mucus)"))
+					return 1
+	return 0
+
+/mob/living/carbon/monkey/proc/handle_virus_updates()
+	if(status_flags & GODMODE)	return 0	//godmode
+	if(bodytemperature > 406)

```

Comment:
As said in the OP I absolute pathed Life.dm. I'm not the one who originally wrote that